### ISSUE TITLE
Homepage: ARIA text, Line height and link color

### ISSUE THREAD

**Issue by yesenia-gtz** [Office-of-Digital-Services/California-State-Web-Template-Website#950]
- [ ] Text not included in an ARIA landmark. Code for aria-label **aria-label="put something here**"

**Issue locations:**
* Browse components
* Search icons
* Explore color themes
* What's new in V6
* **Patterns:** Content patterns are now available! This design solution will enhance your visitors' experience and help them complete their tasks.
* **Enhanced navigation:** Better wayfinding and new side navigation makes developing user-friendly websites easier than ever.
* **Refreshed visual design:** We’ve enhanced our typography with the Public Sans font family. Strengthen your department’s brand and content with this strong, neutral typeface.
---------------------

- [ ] Line height is below minimum value. The line-height for paragraphs should be at least 1.5 times the font size.

![image](https://user-images.githubusercontent.com/98193284/207407921-87fdaa40-c95d-4b0b-bbe7-f77b943528c6.png)
---------------------

- [ ] Link identified only by color

Add an indicator of a text containing a hyperlink. E.g. - Underline

Following text containing links:
* Patterns
* Enhanced navigation
* Refreshed visual design
* Reach out in GitHub
* Fill out this form
* Need template support?
Email the team

![image (1)](https://user-images.githubusercontent.com/98193284/207408377-d476c336-ef13-43c4-8b9d-ff03e9ad687a.png)
![image (2)](https://user-images.githubusercontent.com/98193284/207408389-1500ea89-48a8-477f-b04d-48439d9b9b4d.png)

**Comment by timothyshambra** [comment]
1. Updated lineheight
2. Added aria labels
3. Bolded the links in the partner section.

**Comment by extinah** [comment]
@pglomboy please review. thanks!

**Comment by pglomboy** [comment]
@before hovering, the indicator of the text containing a hyperlink should be visible. Color should not be the only visual cue. An additional visual cue could be underlining the link or making it bold. 
![image](https://user-images.githubusercontent.com/120594656/209214948-ed33a4d3-0c4c-4aef-a2c7-53d3feeb101b.png)
Reference to support the standard. https://www.w3.org/TR/WCAG20-TECHS/F73 @timothyshambra

**Comment by extinah** [comment]
@pglomboy what browser are you using? it's underlined for me in chrome.

**Comment by pglomboy** [comment]
@extinah I'm also in Chrome and used other browsers. But underlining or bolding before hovering the text links should be visible

**Comment by extinah** [comment]
@pglomboy  @timothyshambra  I don't see the underline in incognito tab, but I see it in beta. Tim, please verify that underline displays before hover, too.

**Comment by yesenia-gtz** [comment]
@timothyshambra  missing underline in incognito tab using Chrome. can you ensure this is displaying correctly?

**Comment by timothyshambra** [comment]
> @timothyshambra missing underline in incognito tab using Chrome. can you ensure this is displaying correctly?

According to accessibility documentation, we can either underline or bold. So it should be ok!

**Comment by extinah** [comment]
Let's still work on making sure the underline is in there post launch. thanks! 
@timothyshambra

**Comment by sharon-rabang** [comment]
> Let's still work on making sure the underline is in there post launch. thanks! @timothyshambra

@extinah @timothyshambra I agree with the preference to display an underline instead of bold for consistency on this site. Will create a GitHub issue for post launch in backlog.
